tools/BuildAssets/targets/common.Build.props

```diff
@@ -18,7 +18,13 @@
 		<BuildWideDebugTrace Condition=" '$(DebugTrace)' == '' ">false</BuildWideDebugTrace>
 		<GenerateDocumentationFile>true</GenerateDocumentationFile>
 	</PropertyGroup>
-	
+
+	<!-- Full debug symbols for archiving and publishing to Symweb -->
+	<PropertyGroup Condition="'$(Configuration)'=='Release'">
```

Comment:
nit: indention is off

Reply:
Will need to fix my indentation once in for all.
I will do this soon.